backend/src/configuration/config.ts

```diff
@@ -108,6 +116,19 @@ const configuration = (): ConfigInterface => {
   };
 };
 
+const getEncryptionConfig = (): EncryptionConfig => {
+  const { REFRESH_TOKEN_BYTES, JWT_SECRET, JWT_LIFETIME } = process.env;
+
+  if (!JWT_SECRET) {
+    throw new Error('Missing JWT_SECRET in env');
+  }
+  return {
+    REFRESH_TOKEN_BYTES: REFRESH_TOKEN_BYTES ? Number(REFRESH_TOKEN_BYTES) : 64,
+    JWT_SECRET,
+    JWT_LIFETIME: JWT_LIFETIME || '5m',
```

Comment:
They shouldn't be optional in encryption config interface, because when these values get accessed their type is "string | undefined" (moreover, when passing in these values to the object we may forget to provide the defaults and ts won't complain). they are optional only in .env

Reply:
If they are required in the encryption config, you should probably throw an Error while initializing the encryption config when some parameter isn't specified...

Reply:
Yes, for those that are required (JWT_SECRET) I throw an error